framework/components/ecl-buttons/buttons/README.md

```diff
@@ -1,42 +0,0 @@
-# Why and how to use this component
-Use buttons to indicate actions a user can take.
-To make a clear distinction between multiple actions, you have to use different visual weight for buttons.
-The button with the strongest visual weight will get more attention.
-We've created multiple variants of the button to suit different situations:
-
-## Call-to-action button
-Use these buttons sparingly on the most important actions like "Apply for funding" or "Watch live streaming".
-Style
-The EC yellow is used for call to actions buttons. Yellow is the most luminous of all the colours of the spectrum. It’s the colour that captures our attention more than any other colour.
-The yellow reinforces the attention and allows us to guide our users through the interface.
-
-## Primary button
-Used for primary actions in case of forms, to initiate a search query, to apply a filter.
-Style
-The EC Blue is used for primary buttons and is the default colour of the pages. It is a saturated colour, which makes it fitted for adding visual weight.
-
-## Default button
-The default button is used for important links on the page.
-Style
-White and EC blue is used for the default button. A secondary action isn’t as important as a primary action so it shouldn’t call as much attention.
-
-## Submit & abort options
-The submit and abort buttons are used to confirm a choice or send information. The abort button ('cancel') is optional.
-Style
-See Primary button.
-
-## Button block
-The button block is a series of default buttons with a fixed width. They link to child pages.
-Style
-See Default button style
-
-# Guidance
```

Comment:
A bit painful to see that we already know it's unfinished work and someone will have to update urgently at worst moment